core/modules/admin_bar/admin_bar.inc

```diff
@@ -784,6 +784,7 @@ function admin_bar_theme_settings() {
     '#title' => t('Enabled components'),
     '#options' => array(
       'admin_bar.icon' => t('Icon menu'),
+      'admin_bar.back_to_site_link' => t('Add Back to site link'),
```

Comment:
All of the other items under "Enabled components" list only the name of the item, so we can remove "Add" here, it should simply be 'Back to site link'.